lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -4732,6 +4732,13 @@ void ScriptContext::ClearForInCaches()
     DebugOnly(forInCacheAllocator.CheckIsAllZero(false));
 }
 
+void ScriptContext::ClearAssignCache()
+{
+    if (Cache()->assignCache)
+    {
+        memset(Cache()->assignCache, 0, Js::Cache::AssignCacheSize);
```

Comment:
* sizeof(EnumeratorCache)

Reply:
actually... i should just use the forInCacheAllocator

---
In reply to: [176248209](https://github.com/Microsoft/ChakraCore/pull/4852#discussion_r176248209) [](ancestors = 176248209)